### ISSUE TITLE
Nuget Package for Windows Phone 8.1

### ISSUE THREAD

**Issue by ravensorb** [oysteinkrog/SQLite.Net-PCL#145]
Are there any plans for a nugget package that supports WP 8.1?

**Comment by oysteinkrog** [comment]
Hi,

In the next/upcoming version, all the nuget packages are combined into one, but for now please try the WinRT platform nuget.

**Comment by ravensorb** [comment]
Good idea -- however it doesn't seem to want to install using that package.

**Comment by oysteinkrog** [comment]
Hmm,

Ok, I've pushed a prerelease/beta of 3.0 to nuget.org now.
You can install/see it in the nuget explores if you enable prerelease to show in the list.
As I mentioned it is all just one nuget now, so you will now need any of the old platform nugets, just install the main one.
Please let me know if it works or not:)

**Comment by oysteinkrog** [comment]
Ping

**Comment by ravensorb** [comment]
I’ll try to get to testing this today/tomorrow .

**Comment by oysteinkrog** [comment]
ping

**Comment by ravensorb** [comment]
So far so good.  Only thing I noticed is that updating the Async library did not update the base one.  Looks like there is a package dependency that needs to be updated?

**Comment by ravensorb** [comment]
Did you see this post

https://forums.xamarin.com/discussion/36600/xamarin-forms-android-project-is-requiring-jetbrains-annotations-pcl328-i-do-not-have-resharper

I am hitting the same issue when testing my Android app.

**Comment by oysteinkrog** [comment]
Hmm ok, that is concerning.
Is it only the android app?
I just manually checked all the assemblies and none of them have any references to the jetbrains assembly!

**Comment by oysteinkrog** [comment]
Prerelase 3.0.2 has a workaround for the android problem.

**Comment by ravensorb** [comment]
Still having a few issues - the Async version will not install right now

![Image of issue]
(http://i.imgur.com/onkpp9n.png)

![Image of issue]
(http://i.imgur.com/Nh0NAk1.png)

**Comment by oysteinkrog** [comment]
Oh, hmm.
Perhaps the dependency has to be to 3.0.2-beta..

**Comment by ravensorb** [comment]
Question, I am testing beta4 on Windows Phone and am getting the following crash

An exception of type 'System.TypeLoadException' occurred in SQLite.Net.DLL but was not handled in user code

Additional information: Could not find Windows Runtime type 'Sqlite.Sqlite3'

Are we missing a dependency?

**Comment by oysteinkrog** [comment]
You need to install the sqlite runtime extension in visual studio I think (yes really stupid/annoying).

**Comment by oysteinkrog** [comment]
@ravensorb Have you had a chance to test my suggestion?
I would like to release the current prerelease nuget as a stable version, but I need to be sure that it works for all platforms.

**Comment by ravensorb** [comment]
So I tried adding the runtime extensions earlier this week and it did not work in my sample application  (I was looking at updating XLabs).  I'll try again this weekend (maybe I missed something)
